### ISSUE TITLE
NestedScrollView stopped working with sticky_grouped_list

### ISSUE THREAD

**Issue by usernamekk** [Dimibe/sticky_grouped_list#15]
I was using this before while it was working => physics: NeverScrollableScrollPhysics() but in sticky_grouped_list, it does not work.

**Comment by usernamekk** [comment]
Still waiting for reply. Thanks

**Comment by Dimibe** [comment]
Hi @usernamekk,
can you explain in more detail what you expect and what the current behaviour is?  I tested the list with the option `physics: NeverScrollableScrollPhysics()` and this results in not being able to scroll the list. This is exactly what I would expect.

**Comment by usernamekk** [comment]
Yes I tested the list with the option physics: NeverScrollableScrollPhysics(), it's not working in my end. I'm using this "physics: NeverScrollableScrollPhysics()" manually as stipulated by flutter. But it's not working when used with your package. Thanks.

**Comment by Dimibe** [comment]
@usernamekk I think you need to investigate the issue further on your on. I can not reproduce any issue on my side.
